File: formationSpringConfigJava/src/main/java/formationSpring/beans/Personne.java
package formationSpring.beans;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.beans.factory.annotation.Qualifier;
import org.springframework.stereotype.Component;

@Component
public class Personne implements IPersonne{

	private String nom;
	@Autowired
	//@Qualifier("ajc")
	private Adresse adresse;
	
	public Personne() {
		System.out.println("construction d'une personne");
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}

	public Adresse getAdresse() {
		return adresse;
	}

	public void setAdresse(Adresse adresse) {
		this.adresse = adresse;
	}
	
	
}

File: formationAngular/src/app/formation/model/personne.ts
export class Personne {
  // private prenom: string | undefined;
  // private nom: string | undefined;

  // public constructor(prenom?: string, nom?: string) {
  //   this.nom = nom;
  //   this.prenom = prenom;
  // }
  public constructor(private _prenom?: string, private _nom?: string) {}

  get prenom(): string | undefined {
    return this._prenom;
  }

  set prenom(value: string | undefined) {
    this._prenom = value;
  }

  get nom(): string | undefined {
    return this._nom;
  }

  set nom(value: string | undefined) {
    this._nom = value;
  }
}
